docs/extend.rst

```diff
@@ -90,6 +90,11 @@ Extensions **must** specify the following options:
 
 - ``version``: Extension API version. Currently ``2``.
 
+Extensions may also specify the following options:
+
+- ``dependencies``: Extensions that the extension depends on. List of strings,
+  where each string is an extension name (see :ref:`extension-format-ref`).
```

Comment:
Fingers crossed this works! We'll find out after merge. :) (I can't enable docs builds for an external PR.)